### DISCUSSION TITLE
How to get started?

### DISCUSSION THREAD

**Discussion by FalconGuts** [community/community#82955] [Programming Help] [answered]
### Body

I want to be a front end dev, 
can someone please give me good sources to learn?
thx 

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by On-Fi** [comment, marked as answer]
The tree main things you need to learn are HTML, CSS and JavaScript.

https://www.w3schools.com/
https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web
Are good startingpoints in my opinion


Good luck in your journey and happy hacking :)

**Reply by FalconGuts** [reply]
Thx Bro

**Comment by FalconGuts** [comment]
Does anybody else has suggestions ?

**Comment by aswanth6000** [comment]
You can begin by learning the basics, such as HTML, CSS, and JavaScript. As a practical approach, consider undertaking projects like creating a personal website or building clones of popular websites such as Twitter and YouTube. These hands-on projects will reinforce your understanding of the fundamentals.

For free learning resources, you can refer to:

MDN Web Docs: https://developer.mozilla.org/en-US/
W3Schools: https://www.w3schools.com/
freeCodeCamp: https://www.freecodecamp.org/

These platforms offer comprehensive guides and tutorials to help you grasp the essentials and apply your knowledge through practical projects. Happy learning

**Reply by FalconGuts** [reply]
Thx Bro
